Add cy.wait(6000) for .btn-succes in file logg_access_out.cy.js

MariuszRozycki opened the pull request:
After considering changes from the teacher and students, I create the final PR

MariuszRozycki committed:
Add cy.wait(6000) for .btn-succes in file logg_access_out.cy.js

MariuszRozycki committed:
Add status badges to README.md

Add status badges to README.md

sulenchy reviewed (changes requested):
@MariuszRozycki I could not run your e2e tests on my local. You need to add how to run the test to the README file.
@sulenchy I've added more details to the README.md about how to initiate tests. If you have time, could you please take a look? I would appreciate it. Everything works when I start the tests. Yesterday, I had to explain to a peer that to run all tests after executing npm start for Sass and live-server, the next step is to open a new terminal. This allows for the command npm test to be entered. Another important note is the waiting time for end-to-end (e2e) tests - developers must wait longer before starting these tests.

MariuszRozycki committed:
Update README.md

More instructions are added to section 'Install and start project'.   Print screens and propertly instructions how to start tests.

MariuszRozycki committed:
Update README.md

Revision of grammatical and spelling issues

MariuszRozycki committed:
Update README.md
